pkgs/by-name/pl/plymouth-blahaj-theme/package.nix

```diff
@@ -0,0 +1,50 @@
+{
+  stdenvNoCC,
+  fetchurl,
+  lib,
+  ...
+}:
+
+stdenvNoCC.mkDerivation rec {
+  pname = "plymouth-blahaj-theme";
+  version = "1.0.0";
+
+  src = fetchurl {
```

Comment:
Please use fetchFromGithub, unless the release tarball significantly differs from the repository content

Reply:
Hi, thanks for your review :)
The release tarball only contains the necessary files to install the plymouth theme. It basically only is the `blåhaj` directory of the git repository. I chose this approach as I wanted to prevent the unnecessary download of stuff like the blender file or the `frames` directory.

Reply:
Then it is better to use the release build :)